lib/filterTeachersByAvailability.js

```diff
@@ -0,0 +1,21 @@
+module.exports = (studentAvailability, teachersArray) =>
```

Comment:
Same thing here, I'd wrap the inner code with brackets:
```
module.exports = (studentAvailability, teachersArray) => {
  // existing body code
};
```

Reply:
Done.